Assets/UI/partialscreenhooks.lua

```diff
@@ -308,6 +311,9 @@ function OnTutorialCloseAll()
   if IsPartialScreenOpen("WorldRankings") then    LuaEvents.PartialScreenHooks_CloseWorldRankings(); end
 end
 
+-- ===========================================================================
+-- CQUI - function needed for CQUI
+-- ===========================================================================
 function OnAddScreenHook(hookInfo:table)
```

Comment:
@Azurency This function is not in the original (pre-patch) game files anymore and seems to be called by CQUI only. Please decide if we still need it; there have been some hooks added with the patch...

Reply:
It was never in any base game file, it's the "Modular Screen" mod by @astog https://forums.civfanatics.com/threads/modular-screens.620112/ That was integrated, maybe you can update the comment to reflect that it's "Modular Screen".